Release notes for 3.4.34 (#603)
opensciencegrid/docs: docs/release/3.4/release-3-4-34.md

OSG Software Release 3.4.34
===========================

> **Release Date**: 2019-08-29

Reviewer: ```suggestion
**Release Date**: 2019-08-29    
**Supported OS Versions:** EL7, EL6
```

Summary of changes
------------------

This release contains:

> -   XCache 1.1.1: ATLAS and CMS XCache; Stash Origin HTTP/S Support

Reviewer: ```suggestion
-   XCache 1.1.1: This release includes packages for ATLAS and CMS XCaches as well as Stash Origin HTTP/S support.
```

-   osg-configure 2.4: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid

> -   osg-xrootd 3.4-4: New meta package for OSG XRootD configuration

Reviewer: ```suggestion
-   OSG XRootD 3.5: A meta-package including common configuration across [standalone](/data/xrootd/install-standalone),
    [storage element](/data/xrootd/install-storage-element), and [caching](/data/stashcache/overview) installations of
    XRootD.
```

> -   xrootd-hdfs 2.1.6: now includes default XRootD configuration in /etc

Reviewer: ```suggestion
-   XRootD HDFS 2.1.6: includes default configuration in `/etc/xrootd/40-xrootd-hdfs.cfg`.
```

> -   xrootd-lcmaps 1.7.4: now includes default XRootD configuration in /etc

Reviewer: ```suggestion
-   XRootD LCMAPS 1.7.4: includes default authorization configuration in `/etc/xrootd/config.d/40-xrootd-lcmaps.cfg`.
    To use the default configuration, uncomment the `# set EnableLcmaps = 1` line in `/etc/xrootd/config.d/10-xrootd-lcmaps.cfg`.
```

    - <https://glideinwms.fnal.gov/doc.prd/factory/configuration.html#singularity>
    - <https://glideinwms.fnal.gov/doc.prd/factory/custom_vars.html#singularity_vars>

1. Upgrades from <= 3.4.0 may require merging `/etc/condor/config.d/*.rpmnew` files and a restart of HTCondor.

1. GlideinWMS >= 3.4.5 uses shared port, requiring only port 9618.
   To ease the transition to shared port, the User Collector secondary collectors and CCBs support both shared and
   separate, individual ports.
   To start using shared port, change the secondary collectors lines and the CCBs lines (if any) in
   `/etc/gwms-frontend/frontend.xml`, changing the address to include the shared port sinful string:

        <collector DN="/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=gwms-frontend.domain" group="default" node="gwms-frontend.domain:9618?sock=collector0-40" secondary="True"/>

   Replacing `gwms-frontend-domain` with the hostname of your GlideinWMS frontend.
   See the [GlideinWMS documentation](https://glideinwms.fnal.gov/doc.prd/components/condor.html#collectors ) for details.

Known Issues
------------

None.

Updating to the new release
---------------------------

> To update to the OSG 3.4 series, please consult the page on [updating between release series](/release/release_series#updating-from-osg-31-32-33-to-33-or-34).

Reviewer: ```suggestion
```

OSG Software Release 3.4.34
===========================

**Release Date**: 2019-08-29    
**Supported OS Versions:** EL7, EL6

Summary of changes
------------------

This release contains:

-   XCache 1.1.1: This release includes packages for ATLAS and CMS XCaches as well as Stash Origin HTTP/S support.

> -   osg-configure 2.4: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid

Reviewer: ```suggestion
-   OSG Configure 2.4: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid
```

Author: Done offline

> -   osg-configure 2.4.1: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid

Reviewer: I'd still like to see the formatting change of `OSG Configure` to match everything else

```suggestion
-   OSG Configure 2.4.1: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid
```

Author: Resolve offline

-   OSG XRootD 3.5: A meta-package including common configuration across [standalone](/data/xrootd/install-standalone),
    [storage element](/data/xrootd/install-storage-element), and [caching](/data/stashcache/overview) installations of
    XRootD.
-   XRootD HDFS 2.1.6: includes default configuration in `/etc/xrootd/40-xrootd-hdfs.cfg`.
-   XRootD LCMAPS 1.7.4: includes default authorization configuration in `/etc/xrootd/config.d/40-xrootd-lcmaps.cfg`.
    To use the default configuration, uncomment the `# set EnableLcmaps = 1` line in `/etc/xrootd/config.d/10-xrootd-lcmaps.cfg`.
-   MyProxy 6.2.4: Remove usage statistics collection support
-   [CCTools 7.0.14](http://ccl.cse.nd.edu/software/): Bug fix release

> -   osg-system-profiler 1.4.3: Remove collection of obsolete information

Reviewer: ```suggestion
-   OSG System Profiler 1.4.3: Remove collection of obsolete information
```
